src/components/Footer.js

```diff
@@ -1,7 +1,22 @@
+import styled from 'styled-components';
+
+const GreenFooter = styled.div`
+	display: flex;
+	position: fixed;
+	bottom: 0;
+	align-items: center;
+	justify-content: center;
+	width: 100%;
+	height: 60px;
+	background-color: #61adad;
+	color: white;
+	font-size: 24px;
+`;
+
 export default function Footer() {
 	return (
-		<footer>
-			<div>© 2022 by Me</div>
-		</footer>
+		<GreenFooter>
+			<div>Find your local expert!</div>
```

Comment:
use a `<p>`tag or a `<span>`for text elements. 
Also you should try to use as less divs as possible. Your Styled component should be a `<p>`, so the extra div is not necessary.